docs/live-migration.md

```diff
@@ -0,0 +1,235 @@
+# Live Migration
+
+## Introduction
+
+The Live Migration feature allows [kubevirt](kubevirt.io) virtual machines to be [live migrated](https://kubevirt.io/user-guide/operations/live_migration/)
+while keeping the established TCP connections alive, and preserving the VM IP configuration.
+These two requirements provide seamless live-migration of a KubeVirt VM using OVN-Kubernetes
+cluster default network.
+
+## Requirements
+
+- KubeVirt >= v1.0.0
```

Comment:
could you refresh my mind into this ? What did you add to kubevirt in this particular release?

Reply:
It's at the PR description:

future kubevirt 1.0:
- https://github.com/kubevirt/kubevirt/pull/9290
- https://github.com/kubevirt/kubevirt/pull/9603